sql/table_test.go

```diff
@@ -170,8 +188,21 @@ func TestMakeTableDescIndexes(t *testing.T) {
 		if err != nil {
 			t.Fatalf("%d: %v", i, err)
 		}
-		if !reflect.DeepEqual(d.index, schema.Indexes[0]) {
-			t.Fatalf("%d: expected %+v, but got %+v", i, d.index, schema.Indexes[0])
+		if !reflect.DeepEqual(d.primary, schema.PrimaryIndex) {
+			t.Fatalf("%d: expected %+v, but got %+v", i, d.primary, schema.PrimaryIndex)
+		}
+		if len(d.indexes) > 0 && !reflect.DeepEqual(d.indexes, schema.Indexes) {
```

Comment:
I was surprised to find out that DeepEqual returns false when presented with two empty structures.